src/events/components/DetailView/detail.less

```diff
@@ -44,6 +51,10 @@
   width: 50%;
 }
 
+.blockGrid > div > p {
```

Comment:
selectors for `.blockGrid` could be written in a more Less-like way?

Not really part of this pull-request, just noticed it while readying through

``` less
.blockGrid {
  display: flex;
  flex-wrap: wrap;
  ustify-content: center;

  > div {
    width: 50%;

    > p {
      text-transform: capitalize;
    }
  }

  > .fullBlock {
    width: 100%;
  }
}
```